composer.json

```diff
@@ -23,7 +23,7 @@
         "cweagans/composer-patches": "^1.6",
         "elife/api-client": "^1.0@dev",
         "elife/api-sdk": "dev-master",
-        "elife/patterns": "dev-master",
+        "elife/patterns": "dev-navigation-changes",
```

Comment:
This was just a transient change for demonstration purposes, so you need to merge [the pattern-library PR](https://github.com/elifesciences/pattern-library/pull/1088) and then revert this change back to 'dev-master' before this can be merged.